### PR TITLE
Typelevel blog concerning recent events / dealing with racism in tech

### PR DESCRIPTION
**Pull request by mtomko**
This blog post is not technical in nature, and I tried to be very careful to use facts and figures, to rely on other authorities, make reasonable/constructive suggestions, and tie my ideas back to Typelevel and the technology industry in general. I did this work as part of time away from my work for the  https://www.shutdownstem.com/ strike on June 10th of this year.

It is unfortunately rather U.S.-centric, and I understand that Typelevel is larger than the United States.

I hope you will consider including it. Many professional organizations, even those which have been less overtly political than Typelevel in the past, have published statements explicitly condemning racism and supporting Black Lives Matter. I do not think I can (or should) speak for Typelevel, but I do feel like some mention of current events is warranted and I felt that I could contribute by putting together something.

Thanks very much for your consideration!

Mark

### CODE REVIEW

File: posts/2020-06-10-confronting-racism.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+The United States Equal Employment Opportunity Commission (EEOC)
+produced a
+[report](https://www.eeoc.gov/special-report/diversity-high-tech)
+showing some sobering statistics about the employment of African
+Americans in the technology sector. In the private industry overall,
+African Americans represented 14.4 percent of the workforce, but in
+the technology sector that number was just 7.4 percent. For
+comparison, according to [recent census
+data](https://www.census.gov/quickfacts/fact/table/US/RHI225218),
+black or African Americans make up approimately 13.4 percent of the
```
Comment (rossabaker) [inline posts/2020-06-10-confronting-racism.md:32]:
```suggestion
black or African Americans make up approximately 13.4 percent of the
```

File: posts/2020-06-10-confronting-racism.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+U.S. population. The problem becomes even worse at the executive
+level: African Americans fill between 2 and 5.3 percent of executive
+roles in technology firms.
+
+A [2016 commissioned
+paper](https://www.nap.edu/read/24926/chapter/14#183) produced by the
+U.S. National Academies of Science found that African American and
+Hispanic people were not underrepresented in terms of computer science
+degrees (about 9.2%), but that they were underreprented in the labor
```
Comment (rossabaker) [inline posts/2020-06-10-confronting-racism.md:41]:
```suggestion
degrees (about 9.2%), but that they were underrepresented in the labor
```

File: posts/2020-06-10-confronting-racism.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+there is no shortage of books and resources to help us. Here are a few
+commonly-cited resources:
+
+* Stamped from the Beginning by [Ibrahim
+  Kendi](https://www.ibramxkendi.com/)
+* How to Be an Anti-Racist by Ibrahim Kendi
+* So You Want to Talk About Race by [Ijeoma
+  Oluo](http://www.ijeomaoluo.com/)
+* White Fragility by [Robin DiAngelo](https://robindiangelo.com/)
+* [Seeing White](https://www.sceneonradio.org/seeing-white/) (podcast)
```
Comment (rossabaker) [inline posts/2020-06-10-confronting-racism.md:135]: I think it's important to link to the authors, but would also consider a link to the materials. This is not a showstopper.

File: posts/2020-06-10-confronting-racism.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+In addition, we need to look around our own communities for
+opportunities to volunteer or contribute. If you cannot march,
+consider donating to organizations supporting the Black Lives Matter
+movement or to bail funds.
+
+
+### Black Lives Matter.
+
+
+### Acknowlegements
```
Comment (rossabaker) [inline posts/2020-06-10-confronting-racism.md:146]:
```suggestion
### Acknowledgments
```

File: posts/2020-06-10-confronting-racism.md (hunk 3 above)
Comment (mtomko) [reply posts/2020-06-10-confronting-racism.md:135]: I can do that. I didn't want to link directly to a business site, but if the authors have pages describing their own works, it does make sense to link to them here.

**Pull request merged**